.markdownlint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "MD013": false,
+  "MD033": false
```

Comment:
Disabled **MD013**(Line length rule) & **MD033**(Inline HTML rule).
It allows us to write longer text and use HTML snippets.